### ISSUE TITLE
Find out what breaks parallel execution

### ISSUE THREAD

**Issue by jpommerening** [aixigo/pdfreactor-loader#7]
When webpack builds mulitple PDFs in parallel, `node-java` throws a weird exception.
We should either find out what causes this, or make sure JVM related things are called sequentially.

**Comment by jpommerening** [comment]
Hmm … it seems the two garbage collectors are fighting over the `LogWriter` proxy. (Well, once you get past the `.some( ... instanceOf( ... ) )` thing by just recording somewhere that the logger is initialized.)

I implemented the colorizing log writer in Java, successfully eliminating the problem.

/cc @alex3683 

Btw. One-and-a-half-liner to compile the Java files:
```sh
docker run -i -t -v $(pwd):/tmp/pdfr -w /tmp/pdfr java:7-alpine \
   javac -cp java $(find java -name "*.java")
```

**Comment by jpommerening** [comment]
Fixed on `master` (`v0.3.0`)
